packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -569,6 +569,21 @@ contract Colosseum is Initializable, Semver {
         emit ChallengeDismissed(_outputIndex, _challenger, block.timestamp);
     }
 
+    /**
+     * @notice Deletes the L2 output root forcefully by the Security Council
+     *         when zk-proofing is not possible due to a bug.
+     *
+     * @param _outputIndex Index of the L2 checkpoint output.
+     */
+    function forceDeleteOutput(uint256 _outputIndex)
+        external
+        onlySecurityCouncil
+        outputNotFinalized(_outputIndex)
+    {
+        // Delete output root.
+        L2_ORACLE.replaceL2Output(_outputIndex, DELETED_OUTPUT_ROOT, SECURITY_COUNCIL);
```

Comment:
Don't we need the logic to move pending bonds of the challenges related to the `_outputIndex` to bonds and delete the challenges?